slirp4netns.1.md

```diff
@@ -90,6 +90,9 @@ specify outbound interface slirp should bind to (ipv6 traffic only)
 `--disable-dns` (since v1.1.0)
 disable built-in DNS (10.0.2.3 by default)
 
+`--hwaddress` (since v1.1.9)
```

Comment:
@giuseppe Do you think we have to bump up version to v1.2.0 for a new flag? I think this PR is trivial enough to keep the version to be v1.1.x.

Reply:
I agree with you, v1.1.x is fine